src/revtrans.jl

```diff
@@ -1,5 +1,3 @@
-struct Unsafe end
```

Comment:
This is a duplicate definition https://github.com/BioJulia/Kmers.jl/blob/5dd8f8ff122e538f200582561defc836a3272046/src/kmer.jl#L15